site/client/pages/Information/templates/Intro.html

```diff
@@ -1,26 +1,23 @@
 <template name="Intro">
   <div id="intro" class="full-screen-container">
-  <div id = "cfp-open">
-    <a href = "https://docs.google.com/forms/d/1qUgKrfhuAOpsmqCE-9blPOyd2m7WjeuaO7qBdDdmARM">Call for speakers is Open!</a>
-  </div>
+    <div id="cfp-open" class="hidden-xs">
+      <a href="https://docs.google.com/forms/d/1qUgKrfhuAOpsmqCE-9blPOyd2m7WjeuaO7qBdDdmARM">Call for speakers is Open!</a>
+    </div>
     <div class="item background-cover" style="background-image: url('{{image}}')">
       <div class="overlay-bg dark"></div>
       <div class="container dark full-centered text-centered ">
 
           <div class="col-xs-10 col-sm-10 col-md-8 col-lg-6">
 
-            <section id="keynote" class="content">
-              <img class="img-responsive center-block" src="images/logo_large-480.png" alt="Empire Node Large Gorilla Logo">
-            </section>
+            <h2>EmpireNode 2015</h2>
 
-            <h1>What the fork is going on with Node.js!?</h1>
+            <h1>What the <i>fork</i> is going on with Node.js!?</h1>
```

Comment:
I like it - I think that it emphasizes that it was done on purpose and in good humor.